nengo_viz/static/viz_value.js

```diff
@@ -27,8 +27,10 @@ VIZ.Value = function(parent, sim, args) {
     /** scales for mapping x and y values to pixels */
     this.scale_x = d3.scale.linear();
     this.scale_y = d3.scale.linear();
-    this.scale_y.domain([args.miny || -1, args.maxy || 1]);
-    
+    var miny = (args.miny !== null) ? args.miny : -1;
+    var maxy = (args.maxy !== null) ? args.maxy : 1;
```

Comment:
I'm also tempted to wipe this check out completely....  I don't think there are any cases any more where .miny and .maxy aren't set by the caller.